packages/unified-selection/src/unified-selection/Selectable.ts

```diff
@@ -79,6 +79,9 @@ export interface Selectables {
   custom: Map<string, CustomSelectable>;
 }
 
+/** @internal */
+export const TRANSIENT_ELEMENT_CLASSNAME = "/TRANSIENT";
```

Comment:
@grigasp I think this should be exported, made public and used in `presentation-frontend`. What do you think?

Reply:
agree